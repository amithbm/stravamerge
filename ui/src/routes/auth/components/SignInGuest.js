import React from 'react'
import { Button } from 'react-bootstrap'

export default ({signInAsGuest, router}) => {
    return  <Button bsSize="large" block onClick={() => signInAsGuest(router)}>Guest Demo Access</Button>
}
