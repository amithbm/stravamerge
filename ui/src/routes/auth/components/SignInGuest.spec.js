import React from 'react'
import {shallow} from 'enzyme'
import SignInGuest from './SignInGuest'
import toJson from 'enzyme-to-json'

describe('components', () => {
    describe('<SignInGuest />', () => {
        it('renders as expected', () => {
            const component =  shallow(<SignInGuest />)
            const tree = toJson(component)
            expect(tree).toMatchSnapshot()
        })
    })
})