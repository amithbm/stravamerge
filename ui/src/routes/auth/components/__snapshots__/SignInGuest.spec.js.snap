exports[`components <SignInGuest /> renders as expected 1`] = `
<Button
  active={false}
  block={true}
  bsClass="btn"
  bsSize="large"
  bsStyle="default"
  disabled={false}
  onClick={[Function]}>
  Guest Demo Access
</Button>
`;
